src/main/java/org/mule/extension/ftp/api/ftp/FtpFileAttributes.java

```diff
@@ -23,9 +24,12 @@
 public class FtpFileAttributes extends AbstractFileAttributes {
 
   @Parameter
+  @Optional
   private LocalDateTime timestamp;
 
   @Parameter
+  // TODO MULE-XXXXX: Remove redundant 'name' attribute in next major version,
```

Comment:
Keep the mule issues PG-13

Reply:
the purpose of this is UX. Sometimes users just want the name and not the path. This is easier than explaining how to extract it